pkgs/development/libraries/libgpg-error/default.nix

```
@@ -24,6 +24,10 @@ in stdenv.mkDerivation (rec {
     sha256 = "sha256-/AfnD2xhX4xPWQqON6m43S4soelAj45gRZxnRSuSXiM=";
   };
 
+  # 1.42 breaks (some?) cross-compilation (e.g. x86_64 -> aarch64).
+  # Backporting this fix (merged in upstream master but no release cut) by David Michael <fedora.dm0@gmail.com> https://dev.gnupg.org/rE33593864cd54143db594c4237bba41e14179061c
+  patches = lib.optional (stdenv.buildPlatform != stdenv.hostPlatform) ./fix-1.42-cross-compilation.patch;
```

```suggestion
  patches = [
    ./fix-1.42-cross-compilation.patch;
  ];
```
we target staging anyway.